src/store/modules/auth.js

```diff
@@ -27,6 +28,7 @@ async function handleLogin(context, username, password, signup = false) {
     }
     await reconnectSockets();
     // If the response was successful, the user is logged in
+    context.dispatch('setReturningUser');
```

Comment:
How do we feel about calling this action right here in the handleLogin function instead of the created() on the home view?